boot/nxboot/Kconfig

```diff
@@ -64,6 +64,17 @@ config NXBOOT_BOOTLOADER
 
 if NXBOOT_BOOTLOADER
 
+config NXBOOT_COPY_TO_RAM
+	bool "Copy bootable image to RAM before calling board boot-image function"
+	default n
+
+config NXBOOT_RAMSTART
+	hex "Start address in RAM that the application is to be loaded"
+	default 0x0
+	depends on NXBOOT_COPY_TO_RAM
+	---help---
+		This will be board specific.
```

Comment:
Please improve this help, i.e. is there some recommend position in the heap? Maybe including suggestions how to reserve the space in the linker script to avoid other applications to use, etc

Reply:
I'll do my best :-)

Reply:
Added

Reply:
did you forget to force-push?

Reply:
> did you forget to force-push?

Ha - was ticking them off here...perhaps got ahead of myself marking them resolved (which perhaps I should have left to the reviewers?). Just doing the squash and force push. Watch this space :-)